packages/dashboard-core-plugins/src/prop-types/UIPropTypes.js

```diff
@@ -33,12 +33,30 @@ const Link = PropTypes.shape({
 
 const Links = PropTypes.arrayOf(Link);
 
+const User = PropTypes.shape({
+  name: PropTypes.string.isRequired,
+  operateAs: PropTypes.string.isRequired,
+  groups: PropTypes.arrayOf(PropTypes.string).isRequired,
+  permissions: PropTypes.shape({
+    isSuperUser: PropTypes.bool.isRequired,
+    isQueryViewOnly: PropTypes.bool.isRequired,
+    isNonInteractive: PropTypes.bool.isRequired,
+    canUsePanels: PropTypes.bool.isRequired,
+    canCreateDashboard: PropTypes.bool.isRequired,
+    canCreateCodeStudio: PropTypes.bool.isRequired,
+    canCreateQueryMonitor: PropTypes.bool.isRequired,
+    canCopy: PropTypes.bool.isRequired,
+    canDownloadCsv: PropTypes.bool.isRequired,
+  }),
```

Comment:
I think `permissions` should be required
```suggestion
  }).isRequired,
```